lib/kube/proxy/url.go

```diff
@@ -213,7 +213,7 @@ func getResourceFromRequest(req *http.Request, kubeDetails *kubeDetails) (*types
 	if kubeDetails == nil {
 		return nil, apiResource, nil
 	}
-	codecFactory, rbacSupportedTypes := kubeDetails.getClusterSupportedResources()
+	codecFactory, rbacSupportedTypes, _ := kubeDetails.getClusterSupportedResources()
```

Comment:
Why no error checking?

Reply:
We cover it before but added it in case someone inverts the logic